synapse/cmds/hive.py

```diff
@@ -124,11 +125,17 @@ def parsepath(path):
 
     async def _handle_get(self, core, opts):
         path = self.parsepath(opts.path)
+
         valu = await core.getHiveKey(path)
         if valu is None:
             self.printf(f'{opts.path} not present')
             return
-        self.printf(f'{opts.path}: {pprint.pformat(valu)}')
+
+        if opts.json:
+            rend = json.dumps(valu, indent=4)
```

Comment:
`s_common.jssave()` @ https://github.com/vertexproject/synapse/blob/010/synapse/common.py#L345 does the following:
`json.dumps(js, sort_keys=True, indent=2)`.  We should be consistent here.

Reply:
done